force-app/main/default/classes/SOQL.cls

```diff
@@ -63,9 +80,21 @@ public with sharing class SOQL {
      * @return A list of Opportunities with important opportunities fields
      * Update the corresponding test class and testGetClosedWonOpportunityCountByAccount method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+    public static List<Opportunity> GetClosedWonOpportunityCountByAccount (Id accountId) {
+        Opportunity[] myOpps = [SELECT Name,
+                                       StageName,
+                                       CloseDate,
+                                       AccountId,
+                                       Amount 
+                                FROM   Opportunity 
+                                WHERE accountId = :accountId AND StageName = 'Closed Won'];
+        if (myOpps.size()>0) {
+            return myOpps;
+        }
+        else {
+            return null;
+        }
```

Comment:
You don't have to worry about doing these checks or returning null. It is preferable to NOT return null, really.